crates/contract_history/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+pub const fn current_mainnet() -> &'static [u8] {
+    version_2_2_0()
+}
+
+pub const fn current_testnet() -> &'static [u8] {
+    version_2_2_0()
+}
+
+pub const fn version_2_2_0() -> &'static [u8; 566653] {
+    include_bytes!("../archive/signer-2_2_0.wasm")
+}
```

Comment:
why do we need to fix the output size here?

Reply:
We don't, but given that we know the size here I found it nice to be explicit. This will never change, and this way we have the size of the contracts visible in the code. So I think this is strictly better than just converting this to an arbitrary slice pointer.

Reply:
You mean it will never change until we update the contract in testnet/mainnet?